purpl3_rms/src/App.js

```diff
@@ -20,90 +19,106 @@ import ErrorDoc from './components/errorDoc/errorDoc'
 import ComputerPage from './components/computers/computers'
 import PrivateRoute from './components/PrivateRoute'
 import {LinkContainer} from "react-router-bootstrap"
+import { logoutRequest } from './libpurpl3/purpl3API';
+import {ErrorModal} from "./components/modals/errorModal"
+import {ErrorContext} from "./context/errorContext";
+
 
 function App() {
 
   const [loggedIn, setLogin] = useState(false);
+  const [error, setError] = useState(null);
 
   return (
     <div className="App">
+      <ErrorContext.Provider value={[error, setError]}>
+
       
-      <HashRouter>
-        <Switch>
-          <Route exact path="/" render={(props) => (<Login {...props} sendLoginStatus={getLoginStatus} />)}/>
-          <Route>
-            <div className="MenuBar">
-
-              <Navbar bg="dark" expand="lg" variant="dark">
-              <Navbar.Toggle aria-controls="responsive-navbar-nav" />
-              <Navbar.Brand>
+        <HashRouter>
```

Comment:
idk what this looks like but seems you changed the navigation bar probablyyyy

Reply:
Yeah i am not sure what changed either but i think i changed something here so i am too scared to revert it :)